src/pykeen/pipeline/api.py

```diff
@@ -1071,7 +1114,6 @@ def _safe_evaluate(
             metric_results: MetricResults = evaluator.evaluate(
                 model=model,
                 mapped_triples=mapped_triples,
-                additional_filter_triples=training_triples_factory.mapped_triples,
```

Comment:
we build up `evaluation_kwargs['additional_filter_triples']` programmatically now, so it will be there